module/VuFind/src/VuFind/Feature/BulkActionTrait.php

```diff
@@ -113,7 +93,7 @@ protected function getBulkActionExportConfig()
     public function getBulkActionLimit($action)
     {
         if ($action == 'export') {
-            $formats = $this->getBulkActionExport()->getActiveFormats('bulk');
+            $formats = $this->export->getActiveFormats('bulk');
```

Comment:
Maybe worth throwing an exception if export is unset at this stage?

Reply:
As with the comment above I checked other traits. They only state the dependency in the doc comment and do not check them. And in this case we get an exception anyway if we want to call the method on null. A custom exception might only have a better description. I will still add it if you want.

Reply:
Similarly, it's fine to leave this as-is. I think in my mind, there's a difference between properties that the trait expects the composing class to already have, vs. things that are used primarily by the trait. But in reality this distinction is not usually very clear, so it does not make sense to overcomplicate things based on my feelings here.